pkgs/by-name/xm/xmltoman/package.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, perl538Packages
+, perl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xmltoman";
+  version = "0.6";
+
+  src = fetchFromGitHub {
+    owner = "atsb";
+    repo = "xmltoman";
+    rev = version;
+    hash = "sha256-EmFdGIeBEcTY0Pqp7BJded9WB/DaXWcMNWh6aTsZlLg=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    perl
+  ];
+
+  buildInputs = [
+    perl538Packages.XMLTokeParser
+  ];
+
+  buildPhase = ''
+    mkdir -p $out/usr/local/bin
+    mkdir -p $out/usr/local/man/man1
+    substituteInPlace $src/Makefile --replace-fail "install -D -m 0644 xmltoman.1" "#install -D -m xmltoman.1"
+    substituteInPlace $src/Makefile --replace-fail "install -D -m 0644 xmlmantohtml.1" "#install -D -m xmlmantohtml.1"
+    substituteInPlace $src/Makefile --replace-fail "./xmltoman" "#./xmltoman"
+    substituteInPlace $src/Makefile --replace-fail "prefix=/usr/local" "prefix=$out/usr/local"
+    chmod 0755 $src/xmltoman
+    chmod 0755 $src/xmlmantohtml
+    cd $src
+    ${perl}/bin/perl xmltoman xml/xmltoman.1.xml > $out/usr/local/man/man1/xmltoman.1.gz
```

```suggestion
    perl xmltoman xml/xmltoman.1.xml > $out/usr/local/man/man1/xmltoman.1.gz
```

Will break cross compilation